source/Halibut.TestUtils.PortForwarder/Halibut.TestUtils.PortForwarder.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Given the plan is to move this to its own solution should the project name reflect that? That way the generated nuget package name won't change when it's split from Halibut and create confusion with 2 different port forwarders in different nuget packages.

Reply:
Renamed